_content/monographs/in-praise-of-buddha’s-acts_willemen-charles.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Buddhacarita: In Praise of Buddha’s Acts"
+translator: "Charles Willemen"
+external_url: "https://bdkamerica.org/download/1896"
+drive_links:
+  - "https://drive.google.com/file/d/1ogsp86UfZguXkm1Z_qVrd2fCggIxKfX0/view?usp=sharing"
+course: rebirth-stories
```

Comment:
Hmmm...  this probably doesn't belong here huh... 🤔  Where does it belong though...  `canonical-poetry`? 🤔

Reply:
ya, I used canonical-poetry as a tag, but would make sense to use it as the course